source/MaterialXRuntime/RtPort.cpp

```diff
@@ -48,72 +48,62 @@ void RtPort::setValueString(const string& v)
     hnd()->asA<PvtPort>()->setValueString(v);
```

Comment:
The diff in this file turned out weird. Only difference is moving the colorspace/unit methods from RtPort to RtInput.